CLAUDE.md

```
@@ -290,6 +290,24 @@ if (isI18nEnabled()) { ... }
 const message = GlobalConfig.i18n.get('key', params, 'default');
 ```
 
+### Source Folder Naming
+
+When creating new folders in `packages/core/src/`, follow these naming conventions:
+- Use **singular** form (e.g., `handler/` not `handlers/`)
+- Use **kebab-case** (e.g., `my-feature/` not `myFeature/` or `my_feature/`)
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)